vetomint/src/progress.rs

```diff
@@ -40,6 +42,38 @@ pub(super) fn progress(
                 }
             },
             ConsensusEvent::BlockProposalCreated { .. } => return None,
+            ConsensusEvent::BlockProposalReceived {
+                proposal,
+                proposal_round,
+                proposer,
+                round,
+                ..
+            } => {
+                let current_proposer = decide_proposer(round, height_info);
+                if proposer == current_proposer && state.step == ConsensusStep::Propose {
+                    match proposal_round {
+                        Some(vr) => {
+                            if vr < round {
+                                on_4f_favor_prevote_propose(proposal, height_info, state, round, vr)
```

Comment:
could you change the name of this function
so that we can know that this returns something?

Reply:
> where is on-4f-favor-prevote-prevote-step ?
`on_4f_prevote` function is for `on-4f-favor-prevote-prevote-step`

And I feel necesscity of changing the name of function. But I am agonizing between following name of vetoming-spec or changing it with other standard.
Can you provide your opinion? @P1n3tree, @byeongjee